src/test/java/uk/gov/hmcts/reform/judicialbooking/util/ValidationUtilTest.java

```diff
@@ -196,5 +199,51 @@ void validateDates_endYear1970() {
         );
     }
 
+    @Test
+    void validateDates_beginYearNull() {
+        LocalDate endDate = LocalDate.of(2023, 1, 1);
+        Assertions.assertDoesNotThrow(() ->
+                ValidationUtil.validateBeginAndEndDates(null, endDate)
+        );
+    }
+
+    @Test
+    void validateDates_endYearNull() {
+        LocalDate beginDate = LocalDate.of(2022, 1, 1);
+        Assertions.assertDoesNotThrow(() ->
+                ValidationUtil.validateBeginAndEndDates(beginDate, null)
+        );
+    }
+
+    @Test
+    void validateDates_endYearnot1970() {
+        LocalDate beginDate = LocalDate.of(1971, 1, 1);
+        Assertions.assertDoesNotThrow(() ->
```

Comment:
whats make difference between above method and this one. both will cover same functionality.
validateDates_endYearNull and validateDates_endYearnot1970

Reply:
Removed this